src/main/java/ch/heigvd/statique/Statique.java

```diff
@@ -28,7 +30,17 @@ public static void main(String... args) {
 
   @Override
   public Integer call() throws Exception {
-    CommandLine.usage(this, System.out);
+    if(version) {
+      try {
+        String v = Files.readString(Path.of("src/main/resources/about/version.txt"));
```

Comment:
Use this to read a file from resource directory, use this

```suggestion
      try (InputStream inputStream = getClass().getClassLoader().getResourceAsStream("about/version.txt")) {
        String v = new String(inputStream.readAllBytes());
```